packages/components/src/InputDate/InputDate.css.d.ts

```diff
@@ -0,0 +1,6 @@
+declare const styles: {
```

Comment:
don't need this file as well, no?

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/76b87d32cb43beceec905435da979776985e3003